srcpkgs/pipewire/files/pipewire/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+! [ -d /run/pipewire ] && install -m 755 -g pipewire -o pipewire -d /run/pipewire
+umask 002
+export PIPEWIRE_RUNTIME_DIR=/run/pipewire
+export XDG_CONFIG_HOME=/var/lib
```

Comment:
How do you feel about making `PIPEWIRE_RUNTIME_DIR` configurable? Does that make sense?

Reply:
I think there's no reason to make it configurable. It's the same in [upstream's systemd unit](https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/master/src/daemon/systemd/system/pipewire.service.in) (`%t` expands to /run I guess)